### DISCUSSION TITLE
File upload Error 'NoneType' object has no attribute 'encode'

### DISCUSSION THREAD

**Discussion by prasadautomationtesting** [open-webui/open-webui#8059] [Issues]
# CHECK OUT: https://docs.openwebui.com/troubleshooting/rag#5--400-nonetype-object-has-no-attribute-encode

I'm encountering an error when attempting to upload files through UI, as indicated by the following message: '**NoneType object has no attribute "encode"**'. 

To better understand the issue, I've included some additional details below.

**UI Screen print:**

<img width="616" alt="NoneType Error" src="https://github.com/user-attachments/assets/86dc6377-1ab7-47b5-a4ce-5f772c9ae29c">

**UI Resonse:**

<img width="583" alt="UI Response" src="https://github.com/user-attachments/assets/dc58bd32-edc0-4b48-837d-54ca61c0ca35">

**Backend Request:**

<img width="347" alt="Request" src="https://github.com/user-attachments/assets/41679630-3e62-4254-9e33-8bcbe90bf3ad">

**Backend Response:**

<img width="358" alt="Response" src="https://github.com/user-attachments/assets/910cc4e9-dd4c-4dbd-b9c0-a5befaf3311f">

**Platform:** Windows 10
**Browser**: Chrome - 131.0.6778.86 (Official Build) (64-bit)
**Open WebUI:** 0.4.7

**Attachment:**

![image](https://github.com/user-attachments/assets/1971e487-81ae-4022-abf0-9e17174620f3)

Note: I tried after cleaning the browser cookies as well but no luck.

**Comment by Jazzy96** [comment]
I have encountered it as well. v0.4.7

**Comment by Simi5599** [comment]
I had this error. It seems that in my case, the error was caused by the Embedding Model in the Admin Panel. I had to retype the default embedding model and then press the button on the right. After that, the error disappeared.

![Screenshot 2024-12-04 153031](https://github.com/user-attachments/assets/64aa364b-3ea3-40ab-b6c5-eb9d9b587522)

**Reply by psanders** [reply]
This did the trick for me.

**Reply by pdob-git** [reply]
retyping embedding model also haven't solved issue for me

**Comment by prasadautomationtesting** [comment]
@Simi5599 : It works for me too after following your workaround. Thanks!

Nevertheless, keeping the defect opened for the fix.

**Comment by Simi5599** [comment]
I am looking for the source code to see if i can get this to work or maybe the maintainer of the repo will do this before me ;)

**Comment by Simi5599** [comment]
UPDATE: I think i did it!

In the file located in `/backend/open_webui/apps/retrieval/main.py` I made the following corrections:

1. I created a variable `default_embedding_model="sentence-transformers/all-MiniLM-L6-v2"`

2. In the function `save_docs_to_vector_db` I added the following lines at the very beginning:
# If the embedding model is empty i am setting it back to the default value
    if not app.state.config.RAG_EMBEDDING_MODEL or app.state.config.RAG_EMBEDDING_MODEL.strip() == "":
        app.state.config.RAG_EMBEDDING_MODEL=default_embedding_model
        update_embedding_model(app.state.config.RAG_EMBEDDING_MODEL)
        app.state.EMBEDDING_FUNCTION = get_embedding_function(
            app.state.config.RAG_EMBEDDING_ENGINE,
            app.state.config.RAG_EMBEDDING_MODEL,
            app.state.sentence_transformer_ef,
            (
                app.state.config.OPENAI_API_BASE_URL
                if app.state.config.RAG_EMBEDDING_ENGINE == "openai"
                else app.state.config.OLLAMA_BASE_URL
            ),
            (
                app.state.config.OPENAI_API_KEY
                if app.state.config.RAG_EMBEDDING_ENGINE == "openai"
                else app.state.config.OLLAMA_API_KEY
            ),
            app.state.config.RAG_EMBEDDING_BATCH_SIZE,
        )
        log.info(f"Embedding model was empty. Set to default: {default_embedding_model}")

The code basically resets the empty `RAG_EMBEDDING_MODEL` to the default value stored in a variable.

If @tjbck can review this, I can create a PR or let him handle the commit ;)

PROOF:

1. Let's set the Embedding Model to "" to recreate the issue ![Screenshot 2024-12-04 231802](https://github.com/user-attachments/assets/0b6cc0a3-d189-4b63-a8aa-04069084703c)
2. Let's feed the LMM with a .pdf file and ask a question ![Screenshot 2024-12-04 231836](https://github.com/user-attachments/assets/10a69dda-af3d-4282-8c56-b44db026b815)
3. We can observe that the Embedding Model has beed restored to the default value 
![Screenshot 2024-12-04 231849](https://github.com/user-attachments/assets/fc32cb2f-aa4e-4760-bad3-7bfb06951311)

**Comment by i0ntempest** [comment]
Apparantly this can also be triggered if the embedding model could not be downloaded due to bad network conditions.

**Comment by GrayXu** [comment]
Sometimes I encounter this issue even without modifying the embedding model configuration, and it only returns to normal after resetting the vector database.

**Comment by silentoplayz** [comment]
> Sometimes I encounter this issue even without modifying the embedding model configuration, and it only returns to normal after resetting the vector database.

This issue has been possibly fixed on the `dev` branch with this commit: https://github.com/open-webui/open-webui/commit/4b7f0c5be11ef437301f9ffa5c68fc52fbb37516

**Comment by unlucio** [comment]
I just tried the dev branch and got this error :(
<img width="1447" alt="Screenshot 2025-01-12 at 23 58 49" src="https://github.com/user-attachments/assets/b5a48889-d194-4437-9cdb-31d867e0c451" />

**Comment by unlucio** [comment]
The same error shows up when enabling search too (via google PSE):
<img width="1587" alt="Screenshot 2025-01-13 at 02 43 01" src="https://github.com/user-attachments/assets/293b058e-7a85-405d-ba9b-069331739bd2" />
<img width="1015" alt="Screenshot 2025-01-13 at 02 43 44" src="https://github.com/user-attachments/assets/41650e22-b42f-483d-890e-d2d9a912459d" />

**Comment by chancezheng** [comment]
same error when upload file in 'Knowledge' moudle, version 0.5.4

**Comment by zebulonjones** [comment]
Vers. 0.5.5 and 0.5.6 - I had this issue and found that it was fixed by changing the Embedded Model Engine to ollama and then setting the address to my local ollama api address.

![Embedding](https://github.com/user-attachments/assets/db55db0f-7bb8-4d26-8ba6-57aa70b7a9bc)

**Reply by wienans** [reply]
Yes this works for me too, if you use ollama directly, instead of the integration in open webui.

but make sure you finde the GGUF models on hugging face.

**Reply by zebulonjones** [reply]
> Yes this works for me too, if you use ollama directly, instead of the integration in open webui.
> 
> but make sure you finde the GGUF models on hugging face.

Can you explain which GGUF model and why? I am new to this. The reason I ask is because I have another issues using websearch in Openwebui. After changing the embedding engine and some k setting. I can now get DDGO to work but my local Searxng will not work in Openwebui. It was suggested it has to do with embedding settings or engine.

**Reply by wienans** [reply]
I am new too I used Nomic-embed-text because it looks like many use it. But I also have mixed results with RAG currently. I tested DDGO web search and that worked fine for me by asking about weather.

But with the RAG I wanted to answers some Rulebook questions and for that I had massiv problems that the rag provided the correct chunks and tried different embedding and reranking models all without success. So as the rag of open webui is currently simple it might be a problem if the general system.

**Reply by zebulonjones** [reply]
I switched to nomic-embed and it worked for me but RAG function in detail elude me. Have you tried paraphrase-multilingual ? It was actually the first embed engine I found recommended.

**Comment by wienans** [comment]
Same issue as posted by @unlucio for me on 0.5.7.

It started happening after i changed to nomic-ai/nomic-embed-text-v1.5 as embedding model from BAAI/bge-m3.

Switching back to BAAI/bge-m3 resolves the issue again

**Comment by unlucio** [comment]
I jusr updated to v0.5.7, tried to upload a pdf, and this is the result:

```
INFO  [open_webui.routers.files] file.content_type: application/pdf
INFO  [open_webui.routers.retrieval] save_docs_to_vector_db: document Anemia 01.PDF file-be1d5ec5-03e3-4ee5-b8fc-bc7dcb5d4eb9
INFO  [open_webui.routers.retrieval] adding to collection file-be1d5ec5-03e3-4ee5-b8fc-bc7dcb5d4eb9
ERROR [open_webui.routers.retrieval] 'NoneType' object has no attribute 'encode'
Traceback (most recent call last):
  File "/app/backend/open_webui/routers/retrieval.py", line 777, in save_docs_to_vector_db
    embeddings = embedding_function(
                 ^^^^^^^^^^^^^^^^^^^
  File "/app/backend/open_webui/retrieval/utils.py", line 262, in <lambda>
    return lambda query: embedding_function.encode(query).tolist()
                         ^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'encode'
ERROR [open_webui.routers.retrieval] 'NoneType' object has no attribute 'encode'
Traceback (most recent call last):
  File "/app/backend/open_webui/routers/retrieval.py", line 953, in process_file
    raise e
  File "/app/backend/open_webui/routers/retrieval.py", line 926, in process_file
    result = save_docs_to_vector_db(
             ^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/backend/open_webui/routers/retrieval.py", line 799, in save_docs_to_vector_db
    raise e
  File "/app/backend/open_webui/routers/retrieval.py", line 777, in save_docs_to_vector_db
    embeddings = embedding_function(
                 ^^^^^^^^^^^^^^^^^^^
  File "/app/backend/open_webui/retrieval/utils.py", line 262, in <lambda>
    return lambda query: embedding_function.encode(query).tolist()
                         ^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'encode'
ERROR [open_webui.routers.files] 400: 'NoneType' object has no attribute 'encode'
Traceback (most recent call last):
  File "/app/backend/open_webui/routers/retrieval.py", line 953, in process_file
    raise e
  File "/app/backend/open_webui/routers/retrieval.py", line 926, in process_file
    result = save_docs_to_vector_db(
             ^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/backend/open_webui/routers/retrieval.py", line 799, in save_docs_to_vector_db
    raise e
  File "/app/backend/open_webui/routers/retrieval.py", line 777, in save_docs_to_vector_db
    embeddings = embedding_function(
                 ^^^^^^^^^^^^^^^^^^^
  File "/app/backend/open_webui/retrieval/utils.py", line 262, in <lambda>
    return lambda query: embedding_function.encode(query).tolist()
                         ^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'encode'
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/app/backend/open_webui/routers/files.py", line 74, in upload_file
    process_file(request, ProcessFileForm(file_id=id))
  File "/app/backend/open_webui/routers/retrieval.py", line 962, in process_file
    raise HTTPException(
fastapi.exceptions.HTTPException: 400: 'NoneType' object has no attribute 'encode'
ERROR [open_webui.routers.files] Error processing file: be1d5ec5-03e3-4ee5-b8fc-bc7dcb5d4eb9
Collection file-be1d5ec5-03e3-4ee5-b8fc-bc7dcb5d4eb9 does not exist.
INFO:     10.6.9.126:0 - "POST /api/v1/files/ HTTP/1.1" 200 OK
INFO  [open_webui.routers.retrieval] save_docs_to_vector_db: document Anemia 01.PDF 3ac5dd2c-e2e6-48be-9e1f-8c30ab656ee4
INFO  [open_webui.routers.retrieval] adding to collection 3ac5dd2c-e2e6-48be-9e1f-8c30ab656ee4
ERROR [open_webui.routers.retrieval] 'NoneType' object has no attribute 'encode'
Traceback (most recent call last):
  File "/app/backend/open_webui/routers/retrieval.py", line 777, in save_docs_to_vector_db
    embeddings = embedding_function(
                 ^^^^^^^^^^^^^^^^^^^
  File "/app/backend/open_webui/retrieval/utils.py", line 262, in <lambda>
    return lambda query: embedding_function.encode(query).tolist()
                         ^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'encode'
ERROR [open_webui.routers.retrieval] 'NoneType' object has no attribute 'encode'
Traceback (most recent call last):
  File "/app/backend/open_webui/routers/retrieval.py", line 953, in process_file
    raise e
  File "/app/backend/open_webui/routers/retrieval.py", line 926, in process_file
    result = save_docs_to_vector_db(
             ^^^^^^^^^^^^^^^^^^^^^^^
  File "/app/backend/open_webui/routers/retrieval.py", line 799, in save_docs_to_vector_db
    raise e
  File "/app/backend/open_webui/routers/retrieval.py", line 777, in save_docs_to_vector_db
    embeddings = embedding_function(
                 ^^^^^^^^^^^^^^^^^^^
  File "/app/backend/open_webui/retrieval/utils.py", line 262, in <lambda>
    return lambda query: embedding_function.encode(query).tolist()
                         ^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'encode'
Collection file-be1d5ec5-03e3-4ee5-b8fc-bc7dcb5d4eb9 does not exist.
Collection 3ac5dd2c-e2e6-48be-9e1f-8c30ab656ee4 does not exist.
INFO:     10.6.9.126:0 - "POST /api/v1/knowledge/3ac5dd2c-e2e6-48be-9e1f-8c30ab656ee4/file/add HTTP/1.1" 400 Bad Request
```
can please someone help me debug this problem? I have no idea where to look.

**Comment by unlucio** [comment]
after banging my head for a month I fixed my issue: the default Embedding Model is not working for me. That's what was causing all those errors. I'm using one via API now.
Perhaps a better error message might help others not to spend a month on this. 😅

**Comment by thiamata** [comment]
another input.
I am playing around with two types of installations.
one setup with docker compose (updated to 0.5.10) RAG - file uploading - works fine

the other setup is based on an manually installation from scratch. (updated to 0.5.10) trying to upload files ends in:
- retyping the embedding modell only (still got the message: NoneType object has no attribute encode)
- with using 127.0.0.1:11434, better to say moved from Default to ollama in "emb mod eng" - the message switches to NoneType object is not iterable)
-

**Comment by juneleung** [comment]
same issue when I change default emb to BAAI/bge-m3, does anyone know how to fix it? Thanks

**Reply by unlucio** [reply]
weird, that is actually what fixed the problem for me
<img width="595" alt="scr 2025-03-14 at 18 29 26" src="https://github.com/user-attachments/assets/66e118dc-2549-452b-8231-2a74bef5172d" />

**Comment by pdob-git** [comment]
nothing from above works for me. what else can be done?
Do I have to install BAAI first somehow?

Finally I managed to make it work after installing bge-m3 from Ollama using this articles
[https://github.com/open-webui/open-webui/discussions/7598](https://github.com/open-webui/open-webui/discussions/7598)
[https://cuterwrite.top/en/p/integrate-open-webui-ollama-qwen25-local-rag/](https://cuterwrite.top/en/p/integrate-open-webui-ollama-qwen25-local-rag/)

**Reply by unlucio** [reply]
IMHO any working embedding model can solve this error, as long as it works.

I think the actual meaning of the "File upload Error 'NoneType'" error is: "I wasn't able to marshal the given text in embeddings", and the simplest cause/solution is to ensure the embeddings model is working.

this is why [in this comment](https://github.com/open-webui/open-webui/discussions/8059#discussioncomment-12092291) I suggested that perhaps the best solution to this problem is simply a better error message.

**Reply by pdob-git** [reply]
Totally agree. This works with other embedding models too. I tested this also with mxbai-embed.

**Comment by zoldaten** [comment]
<img width="1584" height="608" alt="2025-08-28_11h49_39" src="https://github.com/user-attachments/assets/0c97454b-00d1-4ca9-b146-304e1f1c037d" />
in my case i tried to set model by clicking download button and it gives another error:
"Never be installed".
I found out that was triton issue - i have to install it on Windows with:
"pip install triton-windows"

**Comment by zic04** [comment]
Hello,
I'm having the same problem. It looks like the template downloaded correctly, but even after changing it, I get the same error.

I am on Azure in a web app.

**Reply by zic04** [reply]
<img width="1834" height="1249" alt="image" src="https://github.com/user-attachments/assets/5b012ad4-3f81-4712-848a-e96457f95c78" />

**Reply by zic04** [reply]
I want to know if the model is really downloaded. How can I check it ?

**Reply by zic04** [reply]
I had triton in backend requirement.txt i test new version

not work ^^

**Reply by zic04** [reply]
Do you know its possible to add file --> all-MiniLM-L6-v2 in docker ?

**Reply by zic04** [reply]
log 


/home/LogFiles/2025_08_28_ln1mdlwk0001DN_default_docker.log  (https://***.azurewebsites.net/api/vfs/LogFiles/2025_08_28_ln1mdlwk0001DN_default_docker.log)
2025-08-28T18:56:38.198934356Z 2025-08-28 18:56:38.198 | INFO     | uvicorn.protocols.http.httptools_impl:send:476 - 86.212.154.17:55322:0 - "GET /api/v1/retrieval/config HTTP/1.1" 200

2025-08-28T18:56:40.177837850Z 2025-08-28 18:56:40.177 | INFO     | open_webui.routers.retrieval:update_embedding_config:280 - Updating embedding model: sentence-transformers/all-MiniLM-L6-v2 to sentence-transformers/all-MiniLM-L6-v2
2025-08-28T18:56:40.465212517Z 2025-08-28 18:56:40.465 | INFO     | open_webui.config:save:212 - Saving 'RAG_EMBEDDING_ENGINE' to the database
2025-08-28T18:56:40.479655827Z 2025-08-28 18:56:40.479 | INFO     | open_webui.config:save:212 - Saving 'RAG_EMBEDDING_MODEL' to the database
2025-08-28T18:56:40.495793677Z 2025-08-28 18:56:40.495 | INFO     | sentence_transformers.SentenceTransformer:__init__:219 - Load pretrained SentenceTransformer: /app/backend/data/cache/embedding/models/models--sentence-transformers--all-MiniLM-L6-v2/snapshots/c9745ed1d9f207416be6d2e6f8de32d1f16199bf
2025-08-28T18:56:40.499168176Z 2025-08-28 18:56:40.499 | INFO     | uvicorn.protocols.http.httptools_impl:send:476 - 86.212.154.17:55322:0 - "POST /api/v1/retrieval/embedding/update HTTP/1.1" 200

2025-08-28T18:56:54.851947782Z 2025-08-28 18:56:54.851 | INFO     | uvicorn.protocols.http.httptools_impl:send:476 - 86.212.154.17:55322:0 - "GET /_app/version.json HTTP/1.1" 304

/home/LogFiles/2025_08_28_ln1mdlwk0001DN_default_scm_docker.log  (https://***.azurewebsites.net/api/vfs/LogFiles/2025_08_28_ln1mdlwk0001DN_default_scm_docker.log)

**Reply by zic04** [reply]
When upload file : "/app/backend/open_webui/retrieval/utils.py", line 421, in <lambda> 2025-08-28T19:00:45.812053088Z return lambda query, prefix=None, user=None:

**Reply by zic04** [reply]
Check with ssh :

oot@e9a6ffebef3c:/app/backend# env | grep -E "(SENTENCE_TRANSFORMERS|HF_HOME|RAG_EMBEDDING)"
root@e9a6ffebef3c:/app/backend# ls -la /app/backend/data/cache/embedding/models/
total 40
drwxrwsr-x 1 root root 4096 Aug 28 19:45 .
drwxrwsr-x 1 root root 4096 Aug 28 19:45 ..
drwxrwsr-x 1 root root 4096 Aug 28 19:45 .locks
drwxrwsr-x 1 root root 4096 Aug 28 20:00 models--sentence-transformers--all-MiniLM-L6-v2
drwxrwsr-x 1 root root 4096 Aug 28 19:45 xet
root@e9a6ffebef3c:/app/backend# ls -la /app/backend/data/cache/embedding/models/models--sentence-transformers--all-MiniLM-L6-v2/
total 48
drwxrwsr-x 1 root root 4096 Aug 28 20:00 .
drwxrwsr-x 1 root root 4096 Aug 28 19:45 ..
drwxrwsr-x 1 root root 4096 Aug 28 19:45 .no_exist
drwxrwsr-x 1 root root 4096 Aug 28 20:00 blobs
drwxrwsr-x 1 root root 4096 Aug 28 19:45 refs
drwxrwsr-x 1 root root 4096 Aug 28 19:45 snapshots
root@e9a6ffebef3c:/app/backend# find /app/backend/data/cache/embedding/models/models--sentence-transformers--all-MiniLM-L6-v2/ -type d
/app/backend/data/cache/embedding/models/models--sentence-transformers--all-MiniLM-L6-v2/
/app/backend/data/cache/embedding/models/models--sentence-transformers--all-MiniLM-L6-v2/snapshots
/app/backend/data/cache/embedding/models/models--sentence-transformers--all-MiniLM-L6-v2/snapshots/c9745ed1d9f207416be6d2e6f8de32d1f16199bf
/app/backend/data/cache/embedding/models/models--sentence-transformers--all-MiniLM-L6-v2/snapshots/c9745ed1d9f207416be6d2e6f8de32d1f16199bf/onnx
/app/backend/data/cache/embedding/models/models--sentence-transformers--all-MiniLM-L6-v2/snapshots/c9745ed1d9f207416be6d2e6f8de32d1f16199bf/openvino
/app/backend/data/cache/embedding/models/models--sentence-transformers--all-MiniLM-L6-v2/snapshots/c9745ed1d9f207416be6d2e6f8de32d1f16199bf/1_Pooling
/app/backend/data/cache/embedding/models/models--sentence-transformers--all-MiniLM-L6-v2/.no_exist
/app/backend/data/cache/embedding/models/models--sentence-transformers--all-MiniLM-L6-v2/.no_exist/c9745ed1d9f207416be6d2e6f8de32d1f16199bf
/app/backend/data/cache/embedding/models/models--sentence-transformers--all-MiniLM-L6-v2/refs
/app/backend/data/cache/embedding/models/models--sentence-transformers--all-MiniLM-L6-v2/blobs
root@e9a6ffebef3c:/app/backend# find /app/backend/data/cache/embedding/models/models--sentence-transformers--all-MiniLM-L6-v2/ -name "*.bin" -o -name "*.json" -o -name "*.txt"
/app/backend/data/cache/embedding/models/models--sentence-transformers--all-MiniLM-L6-v2/snapshots/c9745ed1d9f207416be6d2e6f8de32d1f16199bf/modules.json
/app/backend/data/cache/embedding/models/models--sentence-transformers--all-MiniLM-L6-v2/snapshots/c9745ed1d9f207416be6d2e6f8de32d1f16199bf/config.json
/app/backend/data/cache/embedding/models/models--sentence-transformers--all-MiniLM-L6-v2/snapshots/c9745ed1d9f207416be6d2e6f8de32d1f16199bf/config_sentence_transformers.json
/app/backend/data/cache/embedding/models/models--sentence-transformers--all-MiniLM-L6-v2/snapshots/c9745ed1d9f207416be6d2e6f8de32d1f16199bf/sentence_bert_config.json
/app/backend/data/cache/embedding/models/models--sentence-transformers--all-MiniLM-L6-v2/snapshots/c9745ed1d9f207416be6d2e6f8de32d1f16199bf/data_config.json
/app/backend/data/cache/embedding/models/models--sentence-transformers--all-MiniLM-L6-v2/snapshots/c9745ed1d9f207416be6d2e6f8de32d1f16199bf/1_Pooling/config.json
/app/backend/data/cache/embedding/models/models--sentence-transformers--all-MiniLM-L6-v2/.no_exist/c9745ed1d9f207416be6d2e6f8de32d1f16199bf/adapter_config.json

**Reply by rgaricano** [reply]
easier just delete the cached model and redownload it.

**Reply by zic04** [reply]
I found a debugging solution.

# Create a temporary folder
mkdir -p /tmp/manual_model
cd /tmp/manual_model

# Download the files directly with curl
curl -k -L https://huggingface.co/sentence-transformers/all-MiniLM-L6-v2/resolve/main/pytorch_model.bin -o pytorch_model.bin
curl -k -L https://huggingface.co/sentence-transformers/all-MiniLM-L6-v2/resolve/main/tokenizer.json -o tokenizer.json
curl -k -L https://huggingface.co/sentence-transformers/all-MiniLM-L6-v2/resolve/main/vocab.txt -o vocab.txt

Verify that the files have been downloaded correctly.
cd /tmp/manual_model
ls -lh
# Le pytorch_model.bin  ~90MB

Check the contents of the downloaded file.
cd /tmp/manual_model
head -n 5 pytorch_model.bin

For me I have information that this has been blocked by Zscaler.

I think the tool is missing information about the download size alone.
